nrpy/infrastructures/BHaH/tests/BHaH_defines_h_BHaH_defines.c

```diff
@@ -108,8 +108,7 @@ typedef struct __MoL_gridfunctions_struct__ {
   REAL *restrict diagnostic_output_gfs2;
 } MoL_gridfunctions_struct;
 
-#define LOOP_ALL_GFS_GPS(ii)                                                                                                                         \
-  _Pragma("omp parallel for") for (int(ii) = 0; (ii) < Nxx_plus_2NGHOSTS0 * Nxx_plus_2NGHOSTS1 * Nxx_plus_2NGHOSTS2 * NUM_EVOL_GFS; (ii)++)
+#define LOOP_ALL_GFS_GPS(ii) _Pragma("omp parallel for") for (int(ii) = 0; (ii) < Ntot; (ii)++)
```

Comment:
This macro should probably only exist within MoL; I believe it is only used there.